pkg/scheduler/algorithm/priorities/interpod_affinity.go

```diff
@@ -20,16 +20,16 @@ import (
 	"context"
 	"sync/atomic"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/kubernetes/pkg/scheduler/algorithm"
 	"k8s.io/kubernetes/pkg/scheduler/algorithm/predicates"
 	priorityutil "k8s.io/kubernetes/pkg/scheduler/algorithm/priorities/util"
 	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"
+	schedutil "k8s.io/kubernetes/pkg/scheduler/internal/channels"
```

Comment:
what do you think of keeping the original name of the package? rename it to `schedutil` does not sound like a good idea.

Reply:
+1